[2025-05-07T05:40:04.263+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.sync_and_load_mysql scheduled__2025-05-07T05:39:00+00:00 [queued]>
[2025-05-07T05:40:04.272+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.sync_and_load_mysql scheduled__2025-05-07T05:39:00+00:00 [queued]>
[2025-05-07T05:40:04.272+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-07T05:40:04.287+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): sync_and_load_mysql> on 2025-05-07 05:39:00+00:00
[2025-05-07T05:40:04.292+0000] {standard_task_runner.py:60} INFO - Started process 57774 to run task
[2025-05-07T05:40:04.296+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mymind_mongo_to_mysql_etl', 'sync_and_load_mysql', 'scheduled__2025-05-07T05:39:00+00:00', '--job-id', '37739', '--raw', '--subdir', 'DAGS_FOLDER/mymind_etl_dag.py', '--cfg-path', '/tmp/tmpb98k_gtf']
[2025-05-07T05:40:04.298+0000] {standard_task_runner.py:88} INFO - Job 37739: Subtask sync_and_load_mysql
[2025-05-07T05:40:04.354+0000] {task_command.py:423} INFO - Running <TaskInstance: mymind_mongo_to_mysql_etl.sync_and_load_mysql scheduled__2025-05-07T05:39:00+00:00 [running]> on host 52beca4b988b
[2025-05-07T05:40:04.465+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mymind_mongo_to_mysql_etl' AIRFLOW_CTX_TASK_ID='sync_and_load_mysql' AIRFLOW_CTX_EXECUTION_DATE='2025-05-07T05:39:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-07T05:39:00+00:00'
[2025-05-07T05:40:04.466+0000] {mymind_etl_dag.py:301} INFO - Iniciando sincronización y carga a MySQL: 6 usuarios, 15 transcripciones.
[2025-05-07T05:40:04.476+0000] {base.py:83} INFO - Using connection ID 'mysql_mymind_dw' for task execution.
[2025-05-07T05:40:04.487+0000] {mymind_etl_dag.py:310} INFO - Obteniendo IDs actuales desde MySQL...
[2025-05-07T05:40:04.489+0000] {mymind_etl_dag.py:321} INFO - Usuarios a eliminar: 0
[2025-05-07T05:40:04.489+0000] {mymind_etl_dag.py:322} INFO - Transcripciones a eliminar: 0
[2025-05-07T05:40:04.489+0000] {mymind_etl_dag.py:355} INFO - Cargando 6 usuarios en la tabla users...
[2025-05-07T05:40:04.490+0000] {mymind_etl_dag.py:380} INFO - Usuarios cargados exitosamente.
[2025-05-07T05:40:04.491+0000] {mymind_etl_dag.py:393} INFO - Cargando 15 transcripciones en la tabla transcriptions...
[2025-05-07T05:40:04.493+0000] {mymind_etl_dag.py:423} INFO - Transcripciones cargadas exitosamente.
[2025-05-07T05:40:04.494+0000] {mymind_etl_dag.py:430} INFO - Sincronización y carga a MySQL completada con éxito.
[2025-05-07T05:40:04.494+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-07T05:40:04.505+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=mymind_mongo_to_mysql_etl, task_id=sync_and_load_mysql, execution_date=20250507T053900, start_date=20250507T054004, end_date=20250507T054004
[2025-05-07T05:40:04.548+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-07T05:40:04.579+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
